collect_app/src/main/java/org/odk/collect/android/fragments/FormManagerList.java

```diff
@@ -153,13 +156,24 @@ public void onClick(DialogInterface dialog, int i) {
         alertDialog.show();
     }
 
+    @Override
+    public void progressUpdate(int progress, int total) {
```

Comment:
I don't think we need to add the progress update functionality. It is extra complexity that we could add later if we feel we really need it. For the moment the dialog could just have the message `"Deleting..."`.